shmem.c

```diff
@@ -87,7 +87,7 @@ void newOverTimeClient() {
 
 	// Create the shared memory with enough space for the current overTime slots
 	shm_unlink(name);
-	SharedMemory shm = create_shm(name, sizeof(int) * counters->overTime);
+	SharedMemory shm = create_shm(name, (counters->overTime/pagesize + 1)*pagesize*sizeof(int));
```

Comment:
The first part of this computation,
```c
(counters->overTime/pagesize + 1)
```
deliberately uses integer computation (i.e., `15/16 == 0`, `16/16 == 1`) to determine the number of pages to allocate for having enough space for `counters->overTime` entries.